### PR TITLE
Use reusable workflow

### PR DESCRIPTION
**Pull request by koenderks**

### CODE REVIEW

**Comment by vandenman** [comment]
see https://github.com/koenderks/jaspBain/actions/runs/2221044409

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -1,35 +1,18 @@
...
-      - uses: jasp-stats/jasp-actions/setup-test-env@master
+  unit-tests:
 
-      - name: Run unit tests
-        run: source("tests/testthat.R")
-        shell: Rscript {0}
+    uses: jasp-stats/jasp-actions/.github/workflows/unittests.yml@master
+    with:
+      needs_JAGS:   false
+      needs_igraph: false
```
Comment (vandenman) [inline .github/workflows/unittests.yml:18]:
```suggestion
      needs_igraph: true
```

**Review by vandenman — APPROVED** [review]

**Comment by vandenman** [comment]
@koenderks there are some test failures, both on R-4.1.3 and R-4.2.0. Can you verify the ones on R-4.2.0 Windows? It looks like it's always the same 24 test failures.

**Comment by koenderks** [comment]
I will check this in a bit! It is probably due to the fact that these results are based on sampling. I think i also did not test on ubuntu anymore at some point because of this.

**Comment by vandenman** [comment]
@koenderks how are things going with this PR?

**Comment by koenderks** [comment]
No progress, the tests still fail on Windows 4.2.0 only. How shall we proceed with this?

**Comment by vandenman** [comment]
perhaps ignore them for now? We're going to switch to 4.2.1 soon anyway.

**Comment by koenderks** [comment]
@vandenman It seems that even on R 4.2.2 these unit tests refuse to work

**Comment by vandenman** [comment]
is it possible to change the number of samples or so?

we could also increase the tolerance but only on Linux? We'd have to build that into jaspTools though.

**Comment by koenderks** [comment]
I'm close to desperation on this one...The unit tests still fail for both windows and linux and the outcomes deviate much more than can be controlled for with the custom rounding check. We are looking at differences in the 1000's for some BF's. I just skipped all the relevant tests on windows and linux, let me know if you have a better idea.

**Comment by vandenman** [comment]
Let's skip the tests for now. We can always sit together once and see where the differences come from in RStudio and perhaps prep a PR to bain that would solve the platform-dependent problems.

**Pull request merged**